### DISCUSSION TITLE
How to modify configuration in risingwave dev cluster?

### DISCUSSION THREAD

**Discussion by thexiay** [risingwavelabs/risingwave#13072] [Q&A] [answered]
Is there any way to modify the configuration in dev cluster?
I want to debug risingwave dev cluster frontend, but every time i debug it, the heartbeat interval will crash this process, so i want to modify the configuration `heartbeat_interval_ms` to upper to stop this happen, but i cann't find a easy way to modify configuration in risingwave tools.

**Comment by thexiay** [comment]
Now days i modify configuation is in this way：
add a script in prepare-config task of `Makefile.toml`：
```
cat << EOF > "${PREFIX_CONFIG}/risingwave.toml"
[server]
heartbeat_interval_ms = 3600000
EOF
```

Is there an easier way to modify it?😕

**Comment by BugenZhao** [comment, marked as answer]
If I understand correctly, you're referring to customizing the configuration for the cluster started with `./risedev dev`. You can edit the `risedev.yml` to specify a configuration file for each profile. Only the entries that you intend to override need to be present.

https://github.com/risingwavelabs/risingwave/blob/f6678921d35dcafbb70f7c5ce03d26ac7f68cccf/risedev.yml#L18-L20

**Reply by thexiay** [reply]
It did work. It helps me a lot.
